dhis-2/dhis-services/dhis-service-tracker/src/main/java/org/hisp/dhis/tracker/export/event/JdbcEventStore.java

```diff
@@ -1285,7 +1261,7 @@ inner join (select coc.uid, coc.code, coc.name, coc.attributevalues, coc.categor
  coc.categoryoptioncomboid \
 """;
 
-    if (!isSuper(user)) {
+    if (isNotSuperUser(user)) {
```

Comment:
Not sure if this is something we want to address in this PR, but at some point we should drop this user and start using `UserDetails` instead.

Reply:
feel free to do it anytime 👌🏻